docs/man/Dockerfile.5.md

```diff
@@ -69,7 +69,7 @@ A Dockerfile is similar to a Makefile.
   multiple images. Make a note of the last image ID output by the commit before
   each new **FROM** command.
 
-  -- If no tag is given to the **FROM** instruction, latest is assumed. If the
+  -- If no tag is given to the **FROM** instruction, the latest is assumed. If the
```

Comment:
Actually, I don't think this is correct in this context; the `latest` tag (e.g. `debian:latest`) does not have to be the "latest" image/tagged image. I realise this is confusing; perhaps it should be `the 'latest' tag is assumed`, or even `the tag named 'latest' is assumed`.

**Don't** change it yet, perhaps someone else has a better suggestion :smiley: